src/docs/release/breaking-changes/index.md

```diff
@@ -74,3 +75,4 @@ release, and listed in alphabetical order:
 [The Route Transition record and Transition delegate updates]: /docs/release/breaking-changes/route-transition-record-and-transition-delegate
 [ThemeData's accent properties]: /docs/release/breaking-changes/theme-data-accent-properties
 [Material Chip button semantics]: /docs/release/breaking-changes/material-chip-button-semantics
+[Android v1 embedding app and plugin creation deprecation]: /docs/release/breaking-changes/android-v1-embedding-create-deprecation
```

Comment:
nit: what about "Android v1 embedding deprecation" ?

Reply:
It's not a doc about v1 embedding deprecation since there still needs to be a bunch of more steps before it's deprecated. We're just deprecating the creation templates.